Makefile

```diff
@@ -0,0 +1,95 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ==============================================================================
+# Images to Build
+#
+# The images to build. The format is as follows:
+# [ceph release name]@[arch]@[base os name]@[base os registry]@[base os repo]@[base os tag]
+# e.g., luminous@amd64@ubuntu@_@ubuntu@16.04
+#       luminous@arm64@ubuntu@arm64v8@ubuntu@16.04
+
+#TODO: Staging should also stage the architecture, and the Dockerfile "FROM"
+#      replacement should be done from the baseos_reg/baseos_repo:baseos_tag
+#      rather than a __VARIABLE__ file.
+
+FLAVORS_TO_BUILD ?= \
+	luminous@amd64@ubuntu@_@ubuntu@16.04 \
```

Comment:
Should be removed since this is not supported in that PR.

Reply:
There is a lot in the Makefile that needs removed if it's not supported in this PR including all build/clean/push/daemon/...

Reply:
I mean we just need to diff the actual makefile & the original one and put that diff into a separated commit. That helps understanding what are the differences you introduced.